packages/notifier/README.md

```diff
@@ -102,64 +102,63 @@ observeIteration(subscription, observer);
+distributed systems properties, where there is only one producing site but any number of
+consuming sites. The producer is not vulnerable to the consumers; they cannot cause the kit
+to malfunction or prevent the code producing values from making progress. The consumers are
 not vulnerable to each other; one can’t cause other consumers to hang or miss values.
 
-For distributed operation, all the iteration values---non-final values, successful completion 
-value, failure reason---must be `Passable`; values that can somehow be passed between vats. 
+For distributed operation, all the iteration values---non-final values, successful completion
+value, failure reason---must be `Passable`; values that can somehow be passed between vats.
 The rest of this doc assumes all these values are Passable.
 
-The makeNotifierKit() or makeSubscriptionKit() call makes the notifier/updater or 
-publication/subscription pair on the producer's site. As a result, both the `iterationObserver` and 
-the initial `asyncIterable` are on the producer's site. If Producer Paula sends Consumer Bob 
-the `asyncIterable`, Bob receives a possibly remote reference to the asyncIterable. Consumers can 
-be remote from the producer of their consumed content. 
+The makeNotifierKit() or makeSubscriptionKit() call makes the notifier/updater or
+publication/subscription pair on the producer's site. As a result, both the `iterationObserver` and
+the initial `asyncIterable` are on the producer's site. If Producer Paula sends Consumer Bob
+the `asyncIterable`, Bob receives a possibly remote reference to the asyncIterable. Consumers can
+be remote from the producer of their consumed content.
 
 Bob's code above is still correct if he uses this reference directly, since `observeIteration` only
-needs its first argument to be a reference of some sort to an AsyncIterable conveying Passable 
-values. This reference may be a local AsyncIterable, a remote presence of an AsyncIterable, or a 
-local or remote promise for an AsyncIterable. `observeIteration` only sends it eventual messages 
+needs its first argument to be a reference of some sort to an AsyncIterable conveying Passable
+values. This reference may be a local AsyncIterable, a remote presence of an AsyncIterable, or a
+local or remote promise for an AsyncIterable. `observeIteration` only sends it eventual messages
 using `E` (equivalent to the tildot syntax `~.`), and so doesn't care about these differences.
 
-While correct, Bob’s code is sub-optimal. Its distributed systems properties are not terrible, but 
-Bob does better using `getSharableSubscriptionInternals()` (provided by
-SubscriptionKit). This lets Bob make a local AsyncIterable that coordinates better with producer 
-Paula's IterationObserver. 
+While correct, Bob’s code is sub-optimal. Its distributed systems properties are not terrible, but
+Bob does better using `getUpdatesSince()` (provided by
```

Comment:
I'm concerned that `getUpdatesSince` is a homonym and easily confused with `getUpdateSince`.  We need a better name.  `startIterationSince`?

Reply:
`startIterationSince` didn't sound right. Not sure why, but to my ear it seems it should either be `startIterationAfter` or `getIterationSince`. I went with the latter. Also other renamings for consistency.